include/supl_os_client.rst

```diff
@@ -0,0 +1,35 @@
+.. _supl_client:
+
+SUPL client
+###########
+
+The Secure User-Plane Location (SUPL) client module integrates the external SUPL client library into the |NCS|.
+This library can be used to receive A-GPS data over the SUPL protocol.
+
+The SUPL client library is released under a different license than the |NCS|.
```

Comment:
@bjorn-tore-taraldsen Is it a specific license that we can mention here (by name)?

Reply:
This is a proprietary licence that we made to fulfill the licence agreement we had with OSS Nokalva. So it will be hard to identify be a name. Could we add something like this to highlight that this is a different license?
"Therefore, you must download and install it separately to be able to use this module. 
The license will be made available for you to read before you download the library. Please read through it before you accept and start downloading it."